### PR TITLE
Add options for all "Trigger job to run" payload properties

### LINKED ISSUE #4: Support `steps_override`
The use case here is being able to trigger a `run-operation` with specific arguments.

If a PR would be accepted, I would be happy to take a pass at this (with the caveat that I've never contributed to a GHA before)

### LINKED ISSUE #14: feature request: include dbt api "overrides" as optional inputs into request body
Hello!

Was setting up the dbt-cloud-action to kick off our dbt cloud ci/cd job which builds modified artifacts. 
However, we noticed that the dbt native integration has the ability of using the `schema_override` key to set the schema for each PR. 

Was wondering if it was possible to include some of those optional keys in the "trigger job" [request](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Jobs/operation/triggerRun) that this action sends.

Example would just be a string for any / all of the following: 

``` json
{
"cause": "Kicked off from Airflow",
"git_branch": "staging",
"schema_override": "dbt_cloud_pr_123",
"dbt_version_override": "0.18.0",
"target_name_override": "staging",
"timeout_seconds_override": 3000,
"generate_docs_override": true,
"threads_override": 8
}
```

I see you already send the cause so I'm assuming you'd just have to add some optional inputs for the other keys?

### PR DESCRIPTION
**Pull request by chamini2**
- closes #14 
- closes #15 
- closes #5 
- closes #4

Example use: https://github.com/fal-ai/fal_bike_example/actions/runs/3092140572/jobs/5003079606

```
##[debug]{
##[debug]  "cause": "Some cause",
##[debug]  "git_sha": "21bd3f38e6f3ccfa70c17f4d402474d262a01101",
##[debug]  "threads_override": 1,
##[debug]  "generate_docs_override": false,
##[debug]  "steps_override": [
##[debug]    "dbt seed",
##[debug]    "dbt run"
##[debug]  ]
##[debug]}
```

and it is being set as

``` yaml
      - uses: fal-ai/dbt-cloud-action@override-object
        id: dbt_cloud_run
        with:
          dbt_cloud_token: ${{ secrets.DBT_CLOUD_API_TOKEN }}
          dbt_cloud_account_id: ${{ secrets.DBT_CLOUD_ACCOUNT_ID }}
          dbt_cloud_job_id: ${{ secrets.DBT_CLOUD_JOB_ID }}
          interval: 5
          failure_on_error: false

          cause: "Some cause"
          git_sha: ${{ github.sha }}
          threads_override: 1
          generate_docs_override: false
          # notice the `|` for consider it a string
          steps_override: |
            - dbt seed
            - dbt run

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,32 @@ This action lets you trigger a job run on [dbt Cloud](https://cloud.getdbt.com),
...
+- `interval` - The interval between polls in seconds (Default: `30`)
 
-### Optional
-- `cause` - Cause message to use (Default=`"Triggered by a GitHub Action"`)
-- `git_sha` - The git SHA (e.g. commit) to checkout before running dbt Cloud Job. This refers to the state your repository is when running. ([dbt API docs](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Jobs/operation/triggerRun))
-- `interval` - The interval between polls in seconds (Default=`30`)
+### dbt Cloud Job configuration
 
-We recommend passing sensitive variables as GitHub secrets. [Example usage](https://github.com/fal-ai/fal_bike_example/blob/main/.github/workflows/fal_dbt.yml).
+Use any of the [documented options for the dbt API](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Jobs/operation/triggerRun).
```
Comment (mederka) [inline README.md:22]:
```suggestion
Use any of the [documented options for dbt API](https://docs.getdbt.com/dbt-cloud/api-v2#tag/Jobs/operation/triggerRun).
```

**Review by mederka — COMMENTED** [review]
Great job, @chamini2 ! Quick question: are these changes backwards compatible?

**Comment by chamini2** [comment]
> Great job, @chamini2 ! Quick question: are these changes backwards compatible?

Yes!

File: README.md (hunk 1 above)
Comment (chamini2) [reply README.md:22]: Committed because I thought the mistake was the other way around (that I was missing the `the`). So I reverted the commit.

**Review by mederka — APPROVED** [review]

**Pull request merged**